ci/deps/azure-35-compat.yaml

```diff
@@ -1,30 +0,0 @@
-name: pandas-dev
```

Comment:
I think we want to recreate this file on 36 to pin to the lowest support versions of everything

Reply:
Thanks for the info @WillAyd, didn't know that was what we were testing with that build.  I restored it, moved to Python 3.6, and renamed it from `compat` to `minimum_versions`, so it's more obvious for other people what's the build about.

Reply:
Cool thanks! I think that should be pinned to `python 3.6.0` as well instead of allowing the latest `3.6` release to be picked up

Reply:
makes sense, updated